exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -121,6 +121,20 @@ func TestExporter_pushLogsData(t *testing.T) {
 	})
 }
 
+func TestLogsClusterConfig(t *testing.T) {
+	testClusterConfig(t, func(t *testing.T, dsn string, clusterTest clusterTestConfig, fns ...func(*Config)) {
+		exporter := newTestLogsExporter(t, dsn, fns...)
+		clusterTest.verifyConfig(t, exporter.cfg)
+	})
+}
+
+func TestLogsTableEngineConfig(t *testing.T) {
+	testTableEngineConfig(t, func(t *testing.T, dsn string, engineTest tableEngineTestConfig, fns ...func(*Config)) {
+		exporter := newTestLogsExporter(t, dsn, fns...)
```

Comment:
since `newTestLogsExporter` has passed the fns param, we can remove this line: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/420bdba2c55e5cbddafcfc704172a171a2e1e1f4/exporter/clickhouseexporter/exporter_logs_test.go#L124